### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by calofmijuck**
Set Url and Open Chat link in models

Implement views and urls.py

Implement simple search

Use unions to process querysets

### CODE REVIEW

File: backend/server/models.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import re
 
 # Path to default image
-DEFAULT_IMAGE = '../../images/app_logo.png'
+# DEFAULT_IMAGE = '../../images/app_logo.png'
```
Comment (KRKimDongWoo) [inline backend/server/models.py]: What was it stand for?

File: backend/server/models.py [hunk 2]
```diff
@@ -42,12 +42,12 @@ class Meeting(models.Model):
     # contributer - people who opened the meeting with the host
     max_participant = models.IntegerField()
     deadline = models.DateTimeField('meeting deadline')
-    region = models.CharField(max_length=100, default="")
-    photo = models.ImageField(upload_to=pic_folder, default=DEFAULT_IMAGE)
+    region = models.CharField(max_length=100, blank=True)
```
Comment (KRKimDongWoo) [inline backend/server/models.py]: According to Kakao APIs, we need two variables, Latitude and Longitude (i.e. coordinate for location: (33.450705, 126.570677)), and place name (i.e. description of place:(카카오 본사))
Specific location(i.e. 서울특별시 관악구 관악로 1) can be retrieved by coordinate.

File: backend/server/serializers.py [hunk 3]
```diff
@@ -17,9 +17,10 @@ class Meta:
         fields = ('name', 'meeting_set')
 
 class CommentSerializer(serializers.ModelSerializer):
+    nickname = serializers.ReadOnlyField(source='writer.nickname')
     class Meta:
         model = Comment
-        fields = ('id', 'date', 'comment_text', 'parent_meeting', 'writer')
+        fields = ('id', 'date', 'comment_text', 'parent_meeting', 'writer', 'nickname')
```
Comment (KRKimDongWoo) [inline backend/server/serializers.py]: What this nickname field stands for?

**Comment by corvuspink** [comment]
Cool!

File: backend/server/serializers.py (hunk 3 above)
Comment (calofmijuck) [reply backend/server/serializers.py]: Take a look at line 20, it says source='writer.nickname'. It shows the nickname of the writer.

**Pull request merged**